vitest.workspace.ts

```diff
@@ -0,0 +1,47 @@
+// import { defineWorkersConfig, defineWorkspace, defineWorkersProject } from "@cloudflare/vitest-pool-workers/config";
```

Comment:
jest config was running eslint. We should make sure that linting is run when tests are run, at least for CI. This is not the case with this PR